backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxListenerApi.scala

```diff
@@ -157,7 +155,10 @@ class VeloxListenerApi extends ListenerApi with Logging {
     }
 
     // Initial native backend with configurations.
-    val parsed = GlutenConfigUtil.parseConfig(conf.getAll.toMap)
+    var parsed = GlutenConfigUtil.parseConfig(conf.getAll.toMap)
+    if (isDriver && !inLocalMode(conf)) {
+      parsed += (GlutenConfig.COLUMNAR_VELOX_CACHE_ENABLED.key -> "false")
```

Comment:
Would you like to add some comments in code? Or just leave a link to the relevant issue/PR. Thanks.

Reply:
Done, thanks